zerver/migrations/0263_migrate_is_announcement_only.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-01-25 23:47
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def upgrade_stream_post_policy(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Stream = apps.get_model('zerver', 'Stream')
+    Stream.objects.filter(is_announcement_only=False) \
+        .update(stream_post_policy=1)  # STREAM_POST_POLICY_EVERYONE
+    Stream.objects.filter(is_announcement_only=True) \
+        .update(stream_post_policy=2)  # STREAM_POST_POLICY_ADMINS
```

Comment:
Might be prettier to write these as `STREAM_POST_POLICY_EVERYONE = 1` or something and then access it here rather than doing a comment

Reply:
Sure made the change.
Added it like this as i noticed other data migrations following a similar comment pattern.

Reply:
Yeah, want to submit a commit changing the other migrations to do this?   I think it's more readable.  I guess maybe the cleanest version would be to have `Stream.STREAM_POST_POLICY_EVERYONE = 1` as how we write it, so that the code here can just be normal code.

Reply:
On using `Stream.STREAM_POST_POLICY_EVERYONE`, I get the following error:
`AttributeError: type object 'Stream' has no attribute 'STREAM_POST_POLICY_EVERYONE'`.
(I used the `Stream` from `apps.get_model('zerver', 'Stream')`, not as an import)

Sorry, I left it as `STREAM_POST_POLICY_EVERYONE` for now....

Reply:
Just to be clear, what I mean was to effectilvely monkey-patch that attribute in.  The nature of migrations code (which is running as of a potentially old version of the database schema) this way is that they just have the database model, not any additional attributes.